src/Xamarin.Android.Build.Tasks/ILRepack.targets

```diff
@@ -26,6 +26,7 @@
 				Internalize="true"
 				Verbose="true"
 				DebugInfo="true"
+				TargetPlatformDirectory="$([Microsoft.Build.Utilities.ToolLocationHelper]::GetPathToStandardLibraries('.NETFramework', $(TargetFrameworkVersion), ''))"
```

Comment:
@jonathanpeppers why do we need this? I assume it works on a Mac too?

Reply:
never mind, I saw the comment :)

Reply:
Yeah, this fixed my build locally on macOS. It seemed that this ILRepack thing had its own logic to find the `TargetPlatformDirectory` and it wasn't working with mono...